reference/features/dumps.md

```diff
@@ -51,6 +51,15 @@ If you restart MeiliSearch after creating a dump, you will not be able to use th
 
 ## Importing a dump
 
+Dumps in v0.20.0 and below are no longer compatible with the new versions. Before you start importing, check your [MeiliSearch version](/reference/api/version.md#example) and proceed accordingly.
+
+::: note
+We do not recommend using dumps from a new MeiliSearch version to import an older version.
+For example, you **should not import a dump from MeiliSearch v0.22.0 to MeiliSearch v0.21.0.** But importing a dump from MeiliSearch v0.21.0 to MeiliSearch v0.21.0 or higher will work.
```

Comment:
If you accept my suggestion on `learn/advanced/snapshots_vs_dumps.md`, this should be updated accordingly.

Also, be careful with that single line break between 57 and 58: these sentences either belong to the same paragraph and should be separated by a regular space or they are different paragraphs and should be separated by a blank line.